src/reporter/tests/test_timescale_types.py

```diff
@@ -109,25 +110,6 @@ def gen_entity(entity_id: int,
 # Similar to gen_entity in test_timescale_insert module in translators.tests.
 
 
-def insert_entities(entities: Union[List[dict], dict],
```

Comment:
why delete? Isn't this a better implementation? I mean the one in `utils.py` has a couple of issues

* no batching. it makes a call to the notify endpoint for each entity in the list
* no handling of single entity. it looks like it will bomb out if you pass in a dict instead of a list?

Reply:
Hi @c0c0n3 
as per issue #667, I just remove the duplicity in code, There is nothing change in the logic of insert_entities.

if there is issue, can you please suggest me a way to solve this?

Reply:
hi @NEC-Vishal 

> nothing change in the logic of insert_entities

isn't this `insert_entities` you deleted slightly different than the others? see my two points above. Probably this should be the implementation of the function in utils?